### PR TITLE
Correcting the sample inventory for an HA cluster

### PR DESCRIPTION
**Pull request by etsauer**
#### What does this PR do?
updated inventory

#### How should this be manually tested?
```
ansible-playbook -i /root/code/casl-ansible/inventory/s1-ha.casl.example.com.d/inventory/ code/casl-ansible/playbooks/openshift/end-to-end.yml -e openstack_ssh_public_key=my-key
```
#### Is there a relevant Issue open for this?
n/a

#### Who would you like to review this?
cc: @redhat-cop/casl

### CODE REVIEW

File: inventory/s1-ha.casl.example.com.d/inventory/hosts [hunk 1]
```diff
@@ -4,34 +4,45 @@
...
+masters.s1-ha.casl.example.com
 
 [nodes:children]
 masters
-infra_hosts
-node_hosts
-
-[node_hosts]
-[node_hosts:vars]
-openshift_node_labels="{'region': 'primary', 'zone': 'default'}"
```
Comment (tomassedovic) [inline inventory/s1-ha.casl.example.com.d/inventory/hosts]: All the changes in this PR make sense to me, but why do you remove the node labels? The deployment won't work (out of the box) without them, will it?

Comment (etsauer) [inline inventory/s1-ha.casl.example.com.d/inventory/hosts]: @tomassedovic no, this change won't remove the node labels. it actually is just  removing this host group name, as it's no longer used. the node labels get set based on the instance metadata pulled from openstack.

File: inventory/s1-ha.casl.example.com.d/inventory/group_vars/all.yml [hunk 2]
```diff
@@ -5,21 +5,21 @@ ansible_become: True
...
 openstack_default_image_name: "rhel-guest-image-7.3-35.x86_64"
-openstack_default_flavor: "m1.large"
+openstack_default_flavor: "m1.medium"
 openstack_external_network_name: "external"
-openstack_dns_domain: "casl.example.com" 
+openstack_dns_domain: "casl.example.com"
 openstack_subnet_prefix: 192.168.99
-openstack_nameservers: 10.9.48.31
+openstack_nameservers:
+- 10.9.48.31
```
Comment (oybed) [inline inventory/s1-ha.casl.example.com.d/inventory/group_vars/all.yml]: Can we change these to some more generic IPs? -ie: 192.168.1.1 ??

File: inventory/s1-ha.casl.example.com.d/inventory/group_vars/all.yml [hunk 3]
```diff
@@ -5,21 +5,21 @@ ansible_become: True
...
 openstack_num_masters: 3
 openstack_num_nodes: 2
-openstack_num_infra: 2
+openstack_num_infra: 1
 dns_domain: "casl.example.com"
 cluster_id: "casl.example.com"
 public_dns_forwarder: 10.9.48.31
```
Comment (oybed) [inline inventory/s1-ha.casl.example.com.d/inventory/group_vars/all.yml]: Same here - ie 192.168.1.1

**Comment by etsauer** [comment]
@oybed done.

**Comment by oybed** [comment]
LGTM

**Pull request merged**